packages/web-fragments/src/elements/reframed/reframed.ts

```diff
@@ -93,6 +112,20 @@ export function reframed(
 		}
 		alreadyLoaded = true;
 
+		// see web.ts, section "Handle IFrame request from reframed" for more details
+		if (iframe.contentDocument?.title !== 'Web Fragments: reframed') {
+			// TODO: make this a warning and only later an error
+			throw new WebFragmentError(
+				`Reframed IFrame init error!\n` +
```

Comment:
Can we maybe move the string value to some config global?

Reply:
Maybe, what's your goal? Make this string user configurable? Why?

Reply:
I don't intend to make it user-configurable. I know we have tests but you may want to test this somewhere else in the future and spelling matters.  I always prefer to cache strings and refer to those variables all long. Reduces maintenance and offers more guarantees.

Reply:
I agree, however this is currently the only place where we use this string, so I'm going to keep it as is for now and we'll extract the sting when we need to introduce another reframed init error, which while possible is not likely.